src/main/java/org/jenkinsci/plugins/workflow/libs/SCMSourceRetriever.java

```diff
@@ -108,7 +108,8 @@ public SCMSource getScm() {
             catch (InterruptedIOException e) {
                 throw e;
             }
-            catch (IOException e) {
+            // RuntimeException is a generalize case of untyped exceptions such as hudson.plugins.git.GitException
+            catch (RuntimeException | IOException e) {
```

Comment:
At this point I would just catch all subclasses of `Exception`:
```suggestion
            catch (Exception e) {
```

Reply:
@dwnusbaum - you are right (: 
Fixed